### PR TITLE
closable option is added

### PR DESCRIPTION
**Pull request by mudin**
I am using this for fullscreen gallery viewer and I have nothing behind, 
It means I never want to close it.
I noticed that closable option is needed for such cases.

### CODE REVIEW

**Comment by imcvampire** [comment]
LGTM

File: src/App.vue [hunk 1]
```diff
@@ -17,6 +17,7 @@
         :images="images" 
         ref="lightbox"
         :show-caption="true"
+        :closable="false"
```
Comment (imcvampire) [inline src/App.vue]: Can you set it to `true`. I think the demo app should be demonstrate normal use cases

**Comment by imcvampire** [comment]
Hi @mudin, thank you for your contribution. 

1 more request: can you update docs too? Thank you!

**Comment by pexea12** [comment]
By the way, we use `yarn` for package management. Can you use `yarn` instead of `npm`?

File: src/App.vue (hunk 1 above)
Comment (mudin) [reply src/App.vue]: Yes I forgot to change it back! thanks

**Comment by mudin** [comment]
@imcvampire @pexea12 Thank you for your reviews! I updated the code. Please check it once more

**Pull request merged**